src/NuGet.Services.AzureSearch/Analysis/DescriptionCustomAnalyzer.cs

```diff
@@ -23,6 +23,7 @@ public static class DescriptionAnalyzer
                 IdentifierCustomTokenFilter.Name,
                 TokenFilterName.Stopwords,
                 TokenFilterName.Lowercase,
+                TokenFilterName.Truncate
             });
```

Comment:
Could you add a TODO with the bug you opened: [https://github.com/NuGet/NuGetGallery/issues/7045](https://github.com/NuGet/NuGetGallery/issues/7045)

This is perfectly fine to unblock yourself, but it's not the final solution.

Reply:
Same with `PackageIdCustomAnalyzer`

---
In reply to: [276023764](https://github.com/NuGet/NuGet.Services.Metadata/pull/506#discussion_r276023764) [](ancestors = 276023764)

Reply:
I think the spacing split for tokenization is only applicable to the description analyzer, not to the package id analyzer.